src/store.js

```diff
@@ -0,0 +1,21 @@
+import React from 'react'
```

Comment:
This is not where the store `Provider` should be declared and rendered alongside the app!

Should be placed in `_app.tsx`